### PR TITLE
fix(affiliates): fix various issues found during last automation run

### PR DESCRIPTION
**Pull request by daywiss**
Signed-off-by: David Adams <david@umaproject.org>

**Motivation**

https://app.clubhouse.io/uma-project/story/320/fix-minor-bugs-found-in-last-dev-mining-automation-run

**Summary**

- remove unnecessary console logs
- move dev mining payouts to correct folder
- fix dapp mining pr template
- add extra utility functions for interacting with github
- fix ordering of prop merging in some apps

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [x]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [x]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**
This was opened through clubhouse

### CODE REVIEW

File: packages/affiliates/libs/affiliates/utils.js [hunk 1]
```diff
@@ -182,6 +181,33 @@ async function createGithubIssue({ auth, owner = "UMAprotocol", repo = "protocol
...
+async function requestGithubReviewers({ auth, owner, repo, pull_number, reviewers, ...rest }) {
+  assert(auth, "requires github auth credentials");
+  const octokit = new Octokit({
+    auth
+  });
+  return octokit.pulls.requestReviewers({
+    owner,
+    repo,
+    pull_number,
+    reviewers,
```
Comment (pemulis) [inline packages/affiliates/libs/affiliates/utils.js:193]: Will this automatically request a couple of random reviewers from the UMA team?

File: packages/affiliates/payouts/devmining/2021-03-15_2021-03-22_0019.json [hunk 2]
```diff
@@ -6,187 +6,45 @@
...
+      ["0xd60139B287De1408f8388f5f57fC114Fb4B03328", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E", "2.0.1"],
+      ["0x8E51Ad4EeB19693751a9A3E36b8F098D891Ddc7f", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E", "2.0.1"],
+      ["0x4E8d60A785c2636A63c5Bd47C7050d21266c8B43", "0x97990B693835da58A281636296D2Bf02787DEa17", "2.0.1"],
+      ["0x4F1424Cef6AcE40c0ae4fc64d74B734f1eAF153C", "0x97990B693835da58A281636296D2Bf02787DEa17", "2.0.1"],
+      ["0x964Be01cCe200e168c4ba960a764cBEBa8C01200", "0x9a9dcd6b52B45a78CD13b395723c245dAbFbAb71", "2.0.1"],
+      ["0xDB2E7F6655de37822c3020a8988351CC76caDAD5", "0xAd2B08F878C0Be4C4162A72bbF7c0f547F93bD9f", "2.0.1"],
+      ["0x46f5E363e69798a74c8422BFb9EDB63e3FB0f08a", "0x2CA781A4dcA1f37F52696238e70A004746068F3b", "2.0.1"],
+      ["0xb40BA94747c59d076B3c189E3A031547492013da", "0xd332f9Faf23Ba97d9f0f9EE9b3d8798C49Ff269B", "2.0.1"]
     ],
     "fallbackPrices": [
```
Comment (pemulis) [inline packages/affiliates/payouts/devmining/2021-03-15_2021-03-22_0019.json:41]: What are these?

**Review by pemulis — APPROVED** [review]

File: packages/affiliates/payouts/devmining/2021-03-15_2021-03-22_0019.json (hunk 2 above)
Comment (daywiss) [reply packages/affiliates/payouts/devmining/2021-03-15_2021-03-22_0019.json:41]: this was a previous payout record which was saved to the wrong directory and not linted. i moved and linted it

File: packages/affiliates/libs/affiliates/utils.js (hunk 1 above)
Comment (daywiss) [reply packages/affiliates/libs/affiliates/utils.js:193]: its not wired in to do this, but maybe eventually

**Pull request merged**